setup.py

```diff
@@ -25,11 +25,10 @@
     package_data={"fourmat": ("assets/*.*", "assets/.*")},
     install_requires=(
         "click>=7",
-        # Pin these to avoid unplanned messy diffs.
-        "black==20.8b1",
-        "flake8-bugbear==20.1.4",
-        "flake8==3.8.4",
-        "isort~=5.5",
+        "black~=20",
+        "flake8-bugbear~=21",
+        "flake8~=3",
+        "isort~=5",
```

Comment:
can you make them > < ranges?

Reply:
not sure how to interpret the tilde here, it seems that it has meaning only if you specify at least two numbers... https://www.python.org/dev/peps/pep-0440/#compatible-release

Reply:
Hmm, you're right it specifically states to not use it with a "single segment version number". Let me use `<, >`, etc.

Reply:
```suggestion
        "black>=20,<21",
        "flake8-bugbear>=21,<22",
        "flake8>=3,<4",
        "isort>=5,<6",
```